Zend/zend_closures.stub.php

```diff
@@ -12,6 +12,9 @@ function bindTo(?object $newthis, $newscope = UNKNOWN) {}
 
     function call(object $newthis, ...$parameters) {}
 
-    /** @return Closure */
-    function fromCallable(callable $callable) {}
+    /**
+     * @param callable $callable
+     * @return Closure
+     */
+    function fromCallable($callable) {}
```

Comment:
Why this change?

Reply:
When I regenerated all the stubs without this change, a test (closure_from_callable_basic.phpt) was failing with arg info/ZPP mismatch.

Reply:
Hrm, this is actually a bug in the is_callable implementation: https://bugs.php.net/bug.php?id=78770

Reply:
Nice! I'll remove this change then (and thanks for making it possible to generate stubs for only a single directory)

Reply:
I've just removed the changes from the Zend directory.